recipes/unicorn/all/patches/2.0.0.-0001-cmake-msvc-support.patch

```diff
@@ -0,0 +1,11 @@
+--- CMakeLists.txt
```

Comment:
The filename of this patch does not agree with the one in `conandata.yml`.
There is an extra dot after the `2.0.0`.

I wonder why conan/the hooks haven't complained.

Reply:
You are absolutely right. Thank you vey much for your help!